utils/short_ps.sh

```diff
@@ -0,0 +1 @@
+sudo docker ps --format "table {{.Names}}\t{{.Image}}\t{{.Ports}}"
```

Comment:
We wouldn't normally have sudo here, we'd expect the user to run this with sudo - that makes it more explicit that this has eevated permissions.